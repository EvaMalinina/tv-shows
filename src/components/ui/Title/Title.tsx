import * as React from "react";
import { SearchTitle } from "./title.styled";
import { useState } from "react";


const Title = () => {
  const [text] = useState<string>( 'find your movie');

  return (
    <SearchTitle>{ text }</SearchTitle>
  )
};

export default Title;
