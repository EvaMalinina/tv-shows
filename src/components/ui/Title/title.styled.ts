import styled from "styled-components";

export const SearchTitle = styled.h2`
  text-transform: uppercase;
  color: ${props => props.theme.$white};
  font-size: 30px;
`
