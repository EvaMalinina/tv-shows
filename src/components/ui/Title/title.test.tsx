import React from 'react';
import { render } from '@testing-library/react';

import Title from './Title';

describe('Title component', () => {
  it('It should render Title component', () => {
    const { asFragment } = render(<Title/>);
    expect(asFragment()).toMatchSnapshot();
  })
});