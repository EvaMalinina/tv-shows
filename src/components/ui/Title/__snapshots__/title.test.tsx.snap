// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component It should render Title component 1`] = `
<DocumentFragment>
  <h2
    class="sc-bdfBwQ kEYBXB"
  >
    find your movie
  </h2>
</DocumentFragment>
`;
